README.md

```diff
@@ -318,7 +320,92 @@ sectors.name # Industrials
 sectors.performance # 0.00711
 sectors.last_updated # 1533672000437
 ```
-See [#sector-performance](https://iextrading.com/developer/docs/#sector-performance) for detaile documentation or [sectors.rb](lib/iex/resources/sectors.rb) for returned fields.
+See [#sector-performance](https://iextrading.com/developer/docs/#sector-performance) for detailed documentation or [sectors.rb](lib/iex/resources/sectors.rb) for returned fields.
+
+
+### Get IPO Calendar
+
+Fetches a list of IPOs scheduled for the current and next month. Use `today` to get a list of today's IPOs and `upcoming` to get a list of upcoming IPOs. Each scheduled IPO listing is schematically split into separate lists of comprehensive IPO information in `raw_data` with a corresponding abridged summary in `view_data`.
```

Comment:
You did a great job on this PR @marchyoung !

I understand your point adding these two options. Although when I first read `raw_data`, I was expecting to see indeed the raw data coming from the API, but the data is being changed using the lambdas on the IPO class. That said, in favor for backwards compatibility, I would move the `view_data/raw_data` from the response in a different way into one additional method only that returns the data you want.

```ruby
IEX::Resources::IPOCalendar.today
IEX::Resources::IPOCalendar.today_summary
IEX::Resources::IPOCalendar.upcoming
IEX::Resources::IPOCalendar.upcoming_summary
```

So, this way, we assume that calling `.today` would return `raw_data` by default. If a light version of the returned data is requested, then calling `.today_summary` would do what the `view_data` is currently returning.

```ruby
ipos_today = IEX::Resources::IPOCalendar.today
ipos_today.first.market # => 'NASDAQ Global Select'
ipos_today.first.ceo # => 'Stephane Kasriel'
```

@dblock Any thoughts?

Reply:
@rodolfobandeira , @dblock : Thanks for the feedback! I'd be more than happy to make **view_data/raw_data** return the "JSON-to-Hashie" conversion values only and move the lambda-based conversion logic to new **today/upcoming** and **today_summary/upcoming_summary** methods.

Reply:
Right on @rodolfobandeira, we want a collection of first class objects here.